app/services/hearings_creator.rb

```diff
@@ -15,48 +15,49 @@ def call
 private
 
   def push_prosecution_cases
-    hearing[:prosecutionCases]&.each do |prosecution_case_data|
-      prosecution_case_data[:defendants].each do |defendant_data|
-        next if defendant_data[:offences].any? { |offence| offence.dig(:laaApplnReference, :applicationReference)&.start_with?("A", "Z") }
+    hearing_resulted.hearing.prosecution_cases.each do |prosecution_case|
+      prosecution_case.defendants.each do |defendant|
+        next if defendant.offences.any? do |offence|
+          LaaReference.new(maat_reference: offence.laa_reference_application_reference).dummy_maat_reference?
```

Comment:
This feels a bit OTT to me considering the info we need is on one of the parameters we're passing in to instantiate the LaaReference

Reply:
Yeah I know it's not great, but the knowledge of the concept of a dummy reference should live in one class only ideally, and it currently lives in `LaaReference`. Any suggestions?

Reply:
Yeah I agree. The only thought I have is to have a MaatReference class - but it would basically result in exactly the same line of code. 

Maybe one for a future refactor.